### PR TITLE
Feature/tinymce5update 3 releases 5.0.14 to 5.0

### PR DESCRIPTION
**Pull request by lorenzo-pomili**
Related Ticket: 

Description of Changes, it covers:

- Release TinyMCE 5.0.14
- Release TinyMCE 5.0.13
- Release TinyMCE 5.0.9
- Release TinyMCE 5.0.7
- Release TinyMCE 5.0.6
- Release TinyMCE 5.0.5
- Release TinyMCE 5.0.4
- Release TinyMCE 5.0.3
- Release TinyMCE 5.0.2
- Release TinyMCE 5.0.1
- Release TinyMCE 5.0

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/pages/release-notes/release-notes509.adoc [hunk 1]
```diff
@@ -20,29 +20,29 @@ NOTE: TinyMCE 5.0.8 community was not released to enterprise due to issues found
...
 
-For more information on the media plugin refer to the link:{baseurl}/plugins/opensource/media/[documentation].
+For more information on the media plugin refer to the xref:plugins/opensource/media.adoc[documentation].
 
 == Updates and enhancements
 
 === TinyMCE
 
-The new TinyMCE 5.0.9 editor comes with numerous changes to the previous versions. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version509june262019[changelog].
+The new TinyMCE 5.0.9 editor comes with numerous changes to the previous versions. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version509june262019[changelog].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes509.adoc:45]: This xref doesn't resolve as the `version509june262019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version509june262019]]
== 5.0.9 - 2019-06-26
```

File: modules/ROOT/partials/release-notes/deleted.adoc [hunk 2]
```diff
@@ -9,13 +9,13 @@ In TinyMCE 5.0, some configurations have been removed because they are no longer
...
 | https://www.tiny.cloud/docs-4x/configure/editor-appearance/#insert_button_items[`insert_button_items`]
-| Owing to the changes in the menus and removal of the `context` property, `insert_button_items` is not required in TinyMCE 5.0. Refer to the link:{baseurl}/migration-from-4x/#removedsettings[Migration guide] for a workaround.
+| Owing to the changes in the menus and removal of the `context` property, `insert_button_items` is not required in TinyMCE 5.0. Refer to the xref:migration-from-4x.adoc#removedsettings[Migration guide] for a workaround.
 |===
 
 === Removed themes
 
-* link:{baseurl}/migration-from-4x/#inlite[Inlite]
-* link:{baseurl}/migration-from-4x/#modern[Modern]
+* xref:migration-from-4x.adoc#inlite[Inlite]
```
Comment (lnewson) [inline modules/ROOT/partials/release-notes/deleted.adoc:17]: This xref doesn't resolve as the `inlite` id doesn't exist on the relevant header in `migration-from-4x.adoc`. However, it also is broken on production as well. As such this probably needs changing to:
```suggestion
* xref:migration-from-4x.adoc#inlitethemeremoved[Inlite]
```
and then the id would need to be added to the heading. e.g:
```
[[inlitethemeremoved]]
=== Inlite theme removed
```

File: modules/ROOT/partials/release-notes/deleted.adoc [hunk 3]
```diff
@@ -9,13 +9,13 @@ In TinyMCE 5.0, some configurations have been removed because they are no longer
...
+* xref:migration-from-4x.adoc#modern[Modern]
```
Comment (lnewson) [inline modules/ROOT/partials/release-notes/deleted.adoc:18]: This xref doesn't resolve as the `modern` id doesn't exist on the relevant header in `migration-from-4x.adoc`. However, it also is broken on production as well. As such this probably needs changing to:
```suggestion
* xref:migration-from-4x.adoc#modernthemeremoved[Modern]
```
and then the id would need to be added to the heading. e.g:
```
[[modernthemeremoved]]
=== Modern theme removed
```

File: modules/ROOT/partials/release-notes/deleted.adoc [hunk 4]
```diff
@@ -9,13 +9,13 @@ In TinyMCE 5.0, some configurations have been removed because they are no longer
 | Owing to the enhancements to the new inline toolbar behaviour, `fixed_toolbar_container` is not required in TinyMCE 5.0.
 
 | https://www.tiny.cloud/docs-4x/configure/editor-appearance/#insert_button_items[`insert_button_items`]
-| Owing to the changes in the menus and removal of the `context` property, `insert_button_items` is not required in TinyMCE 5.0. Refer to the link:{baseurl}/migration-from-4x/#removedsettings[Migration guide] for a workaround.
+| Owing to the changes in the menus and removal of the `context` property, `insert_button_items` is not required in TinyMCE 5.0. Refer to the xref:migration-from-4x.adoc#removedsettings[Migration guide] for a workaround.
```
Comment (lnewson) [inline modules/ROOT/partials/release-notes/deleted.adoc:12]: This xref doesn't resolve as the `removedsettings` id doesn't exist on the relevant header in `migration-from-4x.adoc`. However, it also is broken on production as well. As such this probably needs changing to:
```suggestion
| Owing to the changes in the menus and removal of the `context` property, `insert_button_items` is not required in TinyMCE 5.0. Refer to the xref:migration-from-4x.adoc#removededitorsettings[Migration guide] for a workaround.
```
and then the id would need to be added to the heading. e.g:
```
[[removededitorsettings]]
==== Removed Editor settings
```

File: modules/ROOT/partials/release-notes/deleted.adoc [hunk 5]
```diff
@@ -34,13 +34,13 @@ https://www.tiny.cloud/docs-4x/demo/custom-toolbar-listbox/[Listbox] is no longe
...
+* xref:plugins/opensource/preview.adoc[Preview]
+* xref:plugins/opensource/template.adoc[Template]
 
-Refer to the link:{baseurl}/migration-from-4x/[Migration Guide], for more information.
+Refer to the xref:migration-from-4x.adoc[Migration Guide], for more information.
 
 === Other removed component configuration settings
 
-Some UI configurations in TinyMCE 5.0 have changed due to an updated UI engine. Please see this link:{baseurl}/migration-from-4x/#otherremovedcomponentconfigurationsettings[section] of the Migration Guide for more information.
+Some UI configurations in TinyMCE 5.0 have changed due to an updated UI engine. Please see this xref:migration-from-4x.adoc#otherremovedcomponentconfigurationsettings[section] of the Migration Guide for more information.
```
Comment (lnewson) [inline modules/ROOT/partials/release-notes/deleted.adoc:46]: This xref doesn't resolve as the `otherremovedcomponentconfigurationsettings` id doesn't exist on the relevant header in `migration-from-4x.adoc`. However, it also is broken on production as well. As such this probably needs changing to:
```suggestion
Some UI configurations in TinyMCE 5.0 have changed due to an updated UI engine. Please see this xref:migration-from-4x.adoc#removeduiconfigurationsettings[section] of the Migration Guide for more information.
```
and then the id would need to be added to the heading. e.g:
```
[[removeduiconfigurationsettings]]
=== Removed UI configuration settings
```

File: modules/ROOT/pages/release-notes/release-notes507.adoc [hunk 6]
```diff
@@ -16,66 +16,66 @@ TinyMCE 5.0.7 release adds improvements to the TinyMCE editor to improve the ove
...
 
-For more information on *Enhanced Media Embed* refer to the full link:{baseurl}/plugins/premium/mediaembed/#mediaembed_inline_styles[documentation].
+For more information on *Enhanced Media Embed* refer to the full xref:plugins/premium/mediaembed.adoc#mediaembed_inline_styles[documentation].
 
 == Updates and enhancements
 
 === TinyMCE
 
-The new TinyMCE 5.0.7 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version507june52019[changelog].
+The new TinyMCE 5.0.7 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version507june52019[changelog].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes507.adoc:61]: This xref doesn't resolve as the `version507june52019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version507june52019]]
== 5.0.7 - 2019-06-05
```

File: modules/ROOT/pages/release-notes/release-notes5014.adoc [hunk 7]
```diff
@@ -14,7 +14,7 @@
 
 The autocompleter API now provides a new `reload` function to allow the autocompleter menu to reload, with potentially more or different results. This could be used for a "More..." menu item that would load more results.
 
-See the autocompleter documentation for information: https://www.tiny.cloud/docs/ui-components/autocompleter/[\https://www.tiny.cloud/docs/ui-components/autocompleter/]
+See the autocompleter documentation for information: https://www.tiny.cloud/docs/ui-components/autocompleter/[https://www.tiny.cloud/docs/ui-components/autocompleter/]
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes5014.adoc:17]: This should be an xref instead and the URL text will need updating.
```suggestion
See the autocompleter documentation for information: xref:ui-components/autocompleter.adoc[UI components - Autocompleter]
```

File: modules/ROOT/pages/release-notes/release-notes5014.adoc [hunk 8]
```diff
@@ -67,7 +67,7 @@ These changes solve bugs with TinyMCE table functionality behaving unexpectedly
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes5014.adoc:67]: There's a link above under the `Bug Fixes` heading that should be a xref. e.g:
```
For a list of bug fixes included in this release, see: xref:changelog.adoc#version5014august192019[Changelog - Version 5.0.14]
```

File: modules/ROOT/pages/release-notes/release-notes5013.adoc [hunk 9]
```diff
@@ -24,14 +24,14 @@ TinyMCE now supports all valid HTML color formats for the `color_map` setting. F
...
-For more information on the `color_map` setting, refer to the link:{baseurl}/configure/content-appearance/#color_map[documentation].
+For more information on the `color_map` setting, refer to the xref:configure/content-appearance.adoc#color_map[documentation].
 
 == Updates and enhancements
 
 === TinyMCE
 
 The updated TinyMCE 5.0.13 editor comes with numerous changes to the previous versions. A comprehensive list of the updates
-and fixed issues is available in the link:{baseurl}/changelog/#version5013august62019[changelog].
+and fixed issues is available in the xref:changelog.adoc#version5013august62019[changelog].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes5013.adoc:34]: This xref doesn't resolve as the `version5013august62019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version5013august62019]]
== 5.0.13 - 2019-08-06
```

File: modules/ROOT/pages/release-notes/release-notes5013.adoc [hunk 10]
```diff
@@ -24,14 +24,14 @@ TinyMCE now supports all valid HTML color formats for the `color_map` setting. F
 
 NOTE: The previous hex code color format without the "#" is still accepted as valid values.
 
-For more information on the `color_map` setting, refer to the link:{baseurl}/configure/content-appearance/#color_map[documentation].
+For more information on the `color_map` setting, refer to the xref:configure/content-appearance.adoc#color_map[documentation].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes5013.adoc:27]: This xref doesn't resolve as the `color_map` id doesn't exist on the relevant header in `text_color.adoc`. e.g:
```
[[color_map]]
=== `color_map`
```

File: modules/ROOT/pages/release-notes/release-notes5013.adoc [hunk 11]
```diff
@@ -24,14 +24,14 @@ TinyMCE now supports all valid HTML color formats for the `color_map` setting. F
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes5013.adoc:24]: The list above isn't rendering correctly as it needs a blank line between the para and list items.

File: modules/ROOT/pages/release-notes/release-notes501.adoc [hunk 12]
```diff
@@ -10,4 +10,4 @@ TinyMCE 5.0.1 release adds improvements to the TinyMCE editor to improve the ove
 
 == Updates and enhancements
 
-The new TinyMCE 5.0.1 editor comes with significant changes to the previous versions. Tiny has created a more configurable and modern editor. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version501february212019[Changelog].
+The new TinyMCE 5.0.1 editor comes with significant changes to the previous versions. Tiny has created a more configurable and modern editor. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version501february212019[Changelog].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes501.adoc:13]: This xref doesn't resolve as the `version501february212019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version501february212019]]
== 5.0.1 - 2019-02-21
```

File: modules/ROOT/pages/release-notes/release-notes502.adoc [hunk 13]
```diff
@@ -12,17 +12,17 @@ TinyMCE 5.0.2 release adds improvements to the TinyMCE editor to improve the ove
...
-For more information on *Checklist* refer to the full link:{baseurl}/plugins/premium/checklist/[documentation].
+For more information on *Checklist* refer to the full xref:plugins/premium/checklist.adoc[documentation].
 
-There is also a demo provided to explore the *Checklist* capabilities link:{baseurl}/demo/checklist/[here].
+There is also a demo provided to explore the *Checklist* capabilities xref:demo/checklist.adoc[here].
 
 == Updates and enhancements
 
-The new TinyMCE 5.0.2 editor comes with significant changes to the previous versions. Tiny has created a more configurable and modern editor. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version502march52019[Changelog].
+The new TinyMCE 5.0.2 editor comes with significant changes to the previous versions. Tiny has created a more configurable and modern editor. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version502march52019[Changelog].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes502.adoc:28]: This xref doesn't resolve as the `version502march52019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version502march52019]]
== 5.0.2 - 2019-03-05
```

File: modules/ROOT/pages/release-notes/release-notes503.adoc [hunk 14]
```diff
@@ -38,10 +38,10 @@ To start using TinyMCE for Swing as your new rich text editor, the first step is
...
 
-For more information on *TinyMCE for Swing* refer to the full link:{baseurl}/integrations/swing/[documentation].
+For more information on *TinyMCE for Swing* refer to the full xref:integrations/swing.adoc[documentation].
 
 Refer to this https://github.com/tinymce/tinymce-swing-codesamples[GitHub repo] for examples on how to use TinyMCE for Swing.
 
 == Updates and enhancements
 
-The new TinyMCE 5.0.3 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version503march192019[Changelog].
+The new TinyMCE 5.0.3 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version503march192019[Changelog].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes503.adoc:47]: This xref doesn't resolve as the `version503march192019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version503march192019]]
== 5.0.3 - 2019-03-19
```

File: modules/ROOT/pages/release-notes/release-notes504.adoc [hunk 15]
```diff
@@ -12,32 +12,32 @@ TinyMCE 5.0.4 release adds improvements to the TinyMCE editor to improve the ove
...
 
 In TinyMCE 5.0.4, when a new sidebar is registered, a corresponding toolbar button for toggling the sidebar open and close is also created using the same name. This button can then be included in the `toolbar` settings by adding the sidebar name to the toolbar options.
 
-For more information on custom sidebar, refer to the link:{baseurl}/ui-components/customsidebar/[documentation]
+For more information on custom sidebar, refer to the xref:ui-components/customsidebar.adoc[documentation]
 
 == Updates and enhancements
 
-The new TinyMCE 5.0.4 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version504april232019[Changelog].
+The new TinyMCE 5.0.4 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version504april232019[Changelog].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes504.adoc:43]: This xref doesn't resolve as the `version504april232019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version504april232019]]
== 5.0.4 - 2019-04-23
```

File: modules/ROOT/pages/release-notes/release-notes505.adoc [hunk 16]
```diff
@@ -12,34 +12,34 @@ TinyMCE 5.0.5 release adds improvements to the TinyMCE editor to improve the ove
...
 
-For more information on configuring the editors `language` and `directionality`, refer to the link:{baseurl}/configure/localization/[documentation].
+For more information on configuring the editors `language` and `directionality`, refer to the xref:configure/localization.adoc[documentation].
 
 == Updates and enhancements
 
 === TinyMCE
 
-The new TinyMCE 5.0.5 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version505may92019[Changelog].
+The new TinyMCE 5.0.5 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version505may92019[Changelog].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes505.adoc:29]: This xref doesn't resolve as the `version505may92019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version505may92019]]
== 5.0.5 - 2019-05-09
```

File: modules/ROOT/pages/release-notes/release-notes506.adoc [hunk 17]
```diff
@@ -18,19 +18,19 @@ TinyMCE's readonly mode has been extended so that it now works with inline edito
...
 
-For more information on the new API, refer to the link:{baseurl}/plugins/opensource/wordcount/#api[documentation].
+For more information on the new API, refer to the xref:plugins/opensource/wordcount.adoc#api[documentation].
 
 == Updates and enhancements
 
 === TinyMCE
 
-The new TinyMCE 5.0.6 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version506may222019[changelog].
+The new TinyMCE 5.0.6 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version506may222019[changelog].
```
Comment (lnewson) [inline modules/ROOT/pages/release-notes/release-notes506.adoc:33]: This xref doesn't resolve as the `version506may222019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version506may222019]]
== 5.0.6 - 2019-05-22
```

**Review by lnewson — CHANGES_REQUESTED** [review]

File: modules/ROOT/partials/release-notes/deleted.adoc (hunk 2 above)
Comment (lorenzo-pomili) [reply modules/ROOT/partials/release-notes/deleted.adoc]: done

File: modules/ROOT/partials/release-notes/deleted.adoc (hunk 3 above)
Comment (lorenzo-pomili) [reply modules/ROOT/partials/release-notes/deleted.adoc]: done

File: modules/ROOT/partials/release-notes/deleted.adoc (hunk 4 above)
Comment (lorenzo-pomili) [reply modules/ROOT/partials/release-notes/deleted.adoc]: done

File: modules/ROOT/partials/release-notes/deleted.adoc (hunk 5 above)
Comment (lorenzo-pomili) [reply modules/ROOT/partials/release-notes/deleted.adoc]: done

File: modules/ROOT/pages/release-notes/release-notes502.adoc (hunk 13 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes502.adoc:28]: done

File: modules/ROOT/pages/release-notes/release-notes503.adoc (hunk 14 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes503.adoc:47]: done

File: modules/ROOT/pages/release-notes/release-notes504.adoc (hunk 15 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes504.adoc:43]: done

File: modules/ROOT/pages/release-notes/release-notes505.adoc (hunk 16 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes505.adoc:29]: done

File: modules/ROOT/pages/release-notes/release-notes506.adoc (hunk 17 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes506.adoc:33]: done

File: modules/ROOT/pages/release-notes/release-notes507.adoc (hunk 6 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes507.adoc:61]: done

File: modules/ROOT/pages/release-notes/release-notes5013.adoc (hunk 9 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes5013.adoc:34]: done

File: modules/ROOT/pages/release-notes/release-notes501.adoc (hunk 12 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes501.adoc:13]: done

File: modules/ROOT/pages/release-notes/release-notes5014.adoc (hunk 7 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes5014.adoc]: done

File: modules/ROOT/pages/release-notes/release-notes5013.adoc (hunk 10 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes5013.adoc:28]: done

File: modules/ROOT/pages/release-notes/release-notes5013.adoc (hunk 11 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/release-notes/release-notes5013.adoc:25]: done

**Review by lnewson — APPROVED** [review]

**Review by metricjs — APPROVED** [review]

**Pull request merged**